pages/vagrant.md

```diff
@@ -62,8 +62,18 @@ You may want to try and issue the above commands on your system, to get familiar
 ```
 
 ## MacOS(X)
-Open your `Terminal`. We assume that [brew](http://brew.sh/) is already installed.
+- Before installing vagrant, we recommend that you designate a new folder to your work at OLE. This is for a number of reasons, however this enables you to be organized and efficient in your work. If everything is in one place, it makes for a more convenient workflow.
```

Comment:
This suggestion on making a new folder can be applied to any user Mac or Windows. I would find a way to place this sentence universally before each OS subsection so that no matter what environment you are working in, people will universally have the same setup.